doc/flatpak-build-export.xml

```diff
@@ -177,6 +177,14 @@
                 </para></listitem>
             </varlistentry>
 
+            <varlistentry>
+                <term><option>--end-of-life=REASON</option></term>
+
+                <listitem><para>
+                    Mark the build as end-of-life.
```

Comment:
We could mention that REASON is a message that will be displayed to the user (depending on what client they use).